test/CMakeLists.txt

```diff
@@ -13,20 +13,19 @@ include(CeleritasAddTest)
 set(CELERITAS_SOURCE_DIR "${PROJECT_SOURCE_DIR}")
 configure_file(testdetail/TestConfig.cc.in TestConfig.cc @ONLY)
 
-add_library(testcel_harness
+celeritas_add_library(testcel_harness
```

Comment:
Since this library doesn't have CUDA code, is it necessary to use the RDC macro?

Also we shouldn't use `celeritas_add_library` since that moves the library build locations and makes them installable, neither of which we want. We would need to add a `celeritas_test_add_library` or something.

Reply:
Indeed it is not necessary (and anyway in this case, for the internal (i.e. not counting the celeritas part), devolves into the regular `add_library`).  However not using it (or more exactly the upcoming `celeritas_test_add_library`) is brittle, as the developer would have to remember to change it if the test gain `Cuda` code and/or if they copy/paste to create a new test that include `Cuda` code.   [Side (as it does not affect the current question) Note: if the library start to depend on 2 independent Cuda libraries, it becomes a Cuda library. In that case 'this' library is in charge of calling `nvcc` for the union of both independent libraries; this is currently properly implement of executable but for libraries I still need to add code (in `target_link_libraries`) to turn the library into a cuda library]

To complete the direct answer to the question, `celeritas_target_link_library` is the only really necessary part if the library does not contain any `.cu` files.

Reply:
OK, so do you think it's easier to define a `celeritas_test_add_library` macro in the `test/CMakeLists.txt` file or add a comment saying that the test libraries shouldn't have cuda code in them? (I don't think we're going to ever compile CUDA code into the test libs.)